doc/source/configuration.rst

```diff
@@ -39,26 +39,32 @@ In order to do so, we are going to setup a new role ``accounting`` a new user
     # For each of the projects, add the user with the accounting role
     openstack role add --user accounting --project <project> accounting
 
-Moreover, this user needs access to Keystone so as to extract the users
+Policy modifications
+--------------------
+The accounting user needs access to Keystone so as to extract the users
 information. In this case, we can can grant the user just the rights for
-listing the users adding the appropriate rules in your
-``/etc/keystone/policy.json`` as follows. Replace the line::
-
-      "identity:list_users": "rule:admin_required",
-
-with::
-
-      "identity:list_users": "rule:admin_required or role:accounting",
-
-Recent Keystone versions leverage a ``/etc/keystone/policy-yaml`` file, if this
-is your case, substitute the line::
-
-   "identity:list_users": "rule:admin_required"
-
-with::
-
-   "identity:list_users": "rule:admin_required or role:accounting"
-
+listing the users adding the appropriate rules in your policy configuration.
+Depending on your configuration, you need to modify the JSON policy file
+(``/etc/keystone/policy.json``) or the YAML policy file (``/etc/keystone/policy-yaml``).
+The modifications in the policy depend on the Keystone version, please ensure
+that you are applying the correct changes as listed in the following table.
+
++-------------+------------------------------------------------------------------------------+
+|  OpenStack  |                                Policy contents                               |
+|   Version   |                                                                              |
++=============+==========+===================================================================+
+| From Stein  | Original | ``“identity:get_user”: “(role:reader and system_scope:all) or     |
+| (>= 15.0.0) |          | (role:reader and token.domain.id:%(target.user.domain_id)s) or    |
+|             |          | user_id:%(target.user.id)s”``                                     |
+|             +----------+-------------------------------------------------------------------+
+|             | Modified | ``“identity:get_user”: “(role:reader and system_scope:all) or     |
+|             |          | (role:reader and token.domain.id:%(target.user.domain_id)s) or    |
+|             |          | user_id:%(target.user.id)s or role:accounting”``                  |
```

Comment:
Is it more simple to just add the role:reader to the accounting user instead of adding a new role?

Reply:
Yes, I will check the new configuration.